x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_results_content.tsx

```diff
@@ -35,20 +36,24 @@ import { useKibanaUiSetting } from '../../../utils/use_kibana_ui_setting';
 import { FirstUseCallout } from './first_use';
 import { AnomaliesResults } from './sections/anomalies';
 import { LogRateResults } from './sections/log_rate';
+import { getMlLinkFormatter } from './sections/helpers/ml_links';
 
 const JOB_STATUS_POLLING_INTERVAL = 30000;
 
 export const AnalysisResultsContent = ({
   sourceId,
   isFirstUse,
+  jobId,
```

Comment:
This component accesses `LogAnalysisJobs.Context` anyway, so how about just retrieving the job ids directly?